en/peripherals/gripper.md

```diff
@@ -0,0 +1,106 @@
+# Gripper Peripherals
+
+![High-load gripper example](../../assets/hardware/grippers/highload_gripper_example.jpg)
+
+Grippers are a commonly used mechanism for UAV delivery applications. There are multiple types of interfaces (e.g. PWM, CAN) for different products in the market.
+
+## Supported Gripper Types
+
+PX4 supports the following Gripper types, check the individual page to learn how to integrate them.
+
+- [Servo Gripper with PWM](gripper_servo.md)
```

Comment:
What about mavlink gripper support? Is it just that you don't have one to test with? What I'd expect here is that if you get the gimbal command and it isn't addressed to the FC it should be forwarded to the component that can handle it.

Reply:
Good point! And that's totally possible with current setup (haven't tested yet) theoretically. I'm not aware of a real MAVLink Gripper I can get off the shelf & test with. I will ask around and let you know if I get to test it :thinking: 

Or if you are aware of it, let me know as well :+1:

Reply:
I don't know of one. I'll think on how we might integrate this.

Reply:
Initially I'll just add a note that it is theoretically supported. Still working on this doc.